de1plus/de1_de1.tcl

```diff
@@ -0,0 +1,221 @@
+#
+# This breaks existing naming convention as de1.tcl already exists
+# as a script to start de1app.tcl
+#
+#
+# These are not in machine.tcl due to apparent assumptions on inclusion order
+#
+
+package provide de1_de1 1.0
+
+package require de1_event 1.0
+package require de1_logging 1.0
+
+namespace eval ::de1::state {
```

Comment:
We need to make the descision if @johnbuckman @decentjohn s statement:
"Keep it as simple as possible, no namespaces, no lambdas, nothing magical"
is still worth keeping. While it might be nice for new devs, namespaces are imho still trivial enough to not overcomplicate our lives.

Reply:
The couple of places I used lambda expressions were to keep the existing method signatures the same. At least with the `lambda` package they end up looking like a "nameless" proc.

```
	[lambda {event_dict} {
		save_this_espresso_to_history \
			[dict get $event_dict previous_state] \
			[dict get $event_dict this_state] \
		} ]
````
reads very similarly to

```
	proc save_this_espresso_to_history_for_event_callback {event_dict} {
		save_this_espresso_to_history \
			[dict get $event_dict previous_state] \
			[dict get $event_dict this_state] \
		} 
```
without leading one to wonder, "Which of these two nearly identically named functions should I be using?" or why the `{} {}` construct exists in 
```
proc save_espresso_rating_to_history {} {
	#unset -nocomplain ::settings(history_saved)
	set ::settings(history_saved) 0
	save_this_espresso_to_history {} {}
}
```

---

At least for me, keeping the code organized by subsystem or function, is helpful. For me, it improves readability, as well as reducing the risk of one bit of code walking over another's globals.